.clang-format

```diff
@@ -91,5 +91,8 @@ SpacesInSquareBrackets: false
 Standard:        Auto
 TabWidth:        8
 UseTab:          Never
+---
+Language:        Json
+DisableFormat:   true
```

Comment:
Why disabling that formatting? I'd expect that this works for the json file, too.